src/main/java/org/sonatype/repository/vgo/internal/proxy/VgoProxyFacetImpl.java

```diff
@@ -89,14 +90,17 @@ protected Content getCachedContent(final Context context) throws IOException {
   protected Content store(final Context context, final Content content) throws IOException {
     VgoAssetKind assetKind = context.getAttributes().require(VgoAssetKind.class);
     TokenMatcher.State matcherState = vgoPathUtils.matcherState(context);
+
+    if (VGO_LIST.equals(assetKind)) {
```

Comment:
Question: Am I right in saying that this is spanning metadata? Therefore would need merging when fetched via the group repository?